<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="2" hostname="LAPTOP-OOUS1A74" failures="2" tests="30" name="Test" time="59.017" errors="0" timestamp="2025-06-26T15:25:50 IST">
  <testcase classname="com.cognizant.base.Base_Test" name="@BeforeClass browserSetup" time="-1.750931710506E9">
    <failure type="org.testng.TestNGException" message="
Parameter &amp;apos;url&amp;apos; is required by BeforeClass on method browserSetup but has not been marked @Optional or defined
in C:\Users\HP\eclipse-workspace\TestNGPracticep1\testng.xml">
      <![CDATA[org.testng.TestNGException: 
Parameter 'url' is required by BeforeClass on method browserSetup but has not been marked @Optional or defined
in C:\Users\HP\eclipse-workspace\TestNGPracticep1\testng.xml
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 22 stack frames]]>
    </failure>
  </testcase> <!-- @BeforeClass browserSetup -->
  <testcase classname="com.cognizant.base.Base_Test" name="@BeforeMethod loginIntoApplication" time="0.0">
    <skipped/>
  </testcase> <!-- @BeforeMethod loginIntoApplication -->
  <testcase classname="com.cognizant.base.Base_Test" name="@AfterMethod logoutFromApplication" time="0.0">
    <skipped/>
  </testcase> <!-- @AfterMethod logoutFromApplication -->
  <testcase classname="com.cognizant.base.Base_Test" name="@AfterClass teardown" time="0.0">
    <skipped/>
  </testcase> <!-- @AfterClass teardown -->
  <testcase classname="com.cognizant.actualTestNGPractise.AfterSuitePractice" name="suiteAnnotationPractice" time="0.005"/>
  <testcase classname="com.cognizant.actualTestNGPractise.DataProviderPrctice" name="loginIntoApllication" time="9.492"/>
  <testcase classname="com.cognizant.actualTestNGPractise.DataProviderPrctice" name="loginIntoApllication" time="10.654"/>
  <testcase classname="com.cognizant.actualTestNGPractise.DataProviderPrctice" name="loginIntoApllication" time="10.828"/>
  <testcase classname="com.cognizant.actualTestNGPractise.DataProviderPrctice" name="loginIntoApllication" time="15.019"/>
  <testcase classname="com.cognizant.actualTestNGPractise.TestLogin" name="loginWithValidCredential" time="4.284"/>
  <testcase classname="com.cognizant.actualTestNGPractise.TestAddToCart" name="testAddToCart" time="0.0">
    <skipped/>
  </testcase> <!-- testAddToCart -->
  <testcase classname="com.cognizant.xmlpractice.ClassB" name="test4" time="0.0"/>
  <testcase classname="com.cognizant.xmlpractice.ClassB" name="test5" time="0.001"/>
  <testcase classname="com.cognizant.xmlpractice.ClassB" name="test6" time="0.0"/>
  <testcase classname="com.cognizant.xmlpractice.ClassC" name="test7" time="0.009"/>
  <testcase classname="com.cognizant.xmlpractice.ClassC" name="test8" time="0.0"/>
  <testcase classname="com.cognizant.xmlpractice.ClassA" name="test1" time="0.0"/>
  <testcase classname="com.cognizant.xmlpractice.ClassA" name="test2" time="0.0"/>
  <testcase classname="com.cognizant.xmlpractice.ClassA" name="test3" time="0.006"/>
  <testcase classname="com.cognizant.actualTestNGPractise.DepedencyPractice" name="addToCart" time="0.03">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.cognizant.actualTestNGPractise.DepedencyPractice.addToCart(DepedencyPractice.java:11)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- addToCart -->
  <testcase classname="com.cognizant.actualTestNGPractise.DepedencyPractice" name="removeProductFromCart" time="0.001">
    <skipped/>
  </testcase> <!-- removeProductFromCart -->
  <testcase classname="com.cognizant.actualTestNGPractise.InvocationCountPractice" name="testA" time="0.0"/>
  <testcase classname="com.cognizant.actualTestNGPractise.InvocationCountPractice" name="testA" time="0.002"/>
  <testcase classname="com.cognizant.actualTestNGPractise.InvocationCountPractice" name="testA" time="0.0"/>
  <testcase classname="com.cognizant.actualTestNGPractise.PriorityPracitce" name="testC" time="0.011"/>
  <testcase classname="com.cognizant.actualTestNGPractise.PriorityPracitce" name="testA" time="0.001"/>
  <testcase classname="com.cognizant.actualTestNGPractise.PriorityPracitce" name="testB" time="0.0"/>
  <testcase classname="practicetestng.Demo" name="testAddToCart" time="0.004"/>
  <testcase classname="practicetestng.Demo" name="testRemoveProductFromCart" time="0.0"/>
  <testcase classname="com.cognizant.actualTestNGPractise.AssertionPractice" name="softassertionPracitce" time="21.154">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	expected [Men&amp;apos;s Shoes, Clothing &amp;amp; Accessories. Nike IN] but found [sairaj]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [Men's Shoes, Clothing & Accessories. Nike IN] but found [sairaj]
at com.cognizant.actualTestNGPractise.AssertionPractice.softassertionPracitce(AssertionPractice.java:41)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- softassertionPracitce -->
  <testcase classname="com.cognizant.actualTestNGPractise.AssertionPractice" name="testA" time="18.262"/>
  <testcase classname="com.cognizant.actualTestNGPractise.AnnoPractice" name="testA" time="0.0"/>
  <testcase classname="com.cognizant.actualTestNGPractise.AnnoPractice" name="testB" time="0.0"/>
  <testcase classname="com.cognizant.actualTestNGPractise.AnnoPractice" name="testC" time="0.0"/>
  <testcase classname="com.cognizant.base.Bast_Test_Practice" name="@AfterSuite closedbConnection" time="0.0">
    <ignored/>
  </testcase> <!-- @AfterSuite closedbConnection -->
  <testcase classname="com.cognizant.base.Bast_Test_Practice" name="@BeforeSuite dbConnection" time="0.0">
    <ignored/>
  </testcase> <!-- @BeforeSuite dbConnection -->
</testsuite> <!-- Test -->
